internal/core/src/exec/expression/UnaryExpr.cpp

```diff
@@ -1961,19 +1990,21 @@ PhyUnaryRangeFilterExpr::ExecNgramMatch() {
         cached_index_chunk_valid_res_ = std::move(valid_res);
     }
 
-    auto real_batch_size =
-        (current_data_chunk_pos_ + batch_size_ > active_count_)
-            ? active_count_ - current_data_chunk_pos_
-            : batch_size_;
+    auto real_batch_size = GetNextBatchSize();
+    if (real_batch_size == 0) {
+        return std::nullopt;
+    }
+
+    TargetBitmap result;
```

Comment:
/go/src/github.com/milvus-io/milvus/internal/core/src/exec/expression/UnaryExpr.cpp:1998:18: error: redeclaration of 'milvus::TargetBitmap result'
 1998 |     TargetBitmap result;
      |                  ^~~~~~